src/AzureRMAssets.ts

```diff
@@ -224,7 +227,7 @@ interface FunctionMetadataContract {
         expectedUsage: string;
         description: string;
         minimumArguments: number;
-        maximumArguments: number;
+        maximumArguments: number | null;
```

Comment:
>number | null; [](start = 26, length = 14)

I'm confused - I thought we were switching from null to undefined? Or does it have to be null here?

Reply:
It has to be null here because JSON doesn't support undefined, and this is the contract for reading the metadata from JSON.